### ISSUE TITLE
Sort playlists by regular list instead of grid

### ISSUE THREAD

**Issue by spriteclad** [omarroth/invidious#452]
I'd like a way to view playlists in a normal list view instead of a grid like it is now. I'd also like the list thumbnails to be a bit smaller than the grid ones, so it's similar to how YT presents their playlist pages.

**Comment by paulaime** [comment]
Agreed, I like better browsing through my subscriptions when it's presented as a list.
FreeTube does that (like YouTube) and it's a great option to have I think.

**Comment by spriteclad** [comment]
Bump.

**Comment by spriteclad** [comment]
Bump. (2)
